Add reproducilibity code for the curvilinear tests

andrewwinters5000 opened the pull request.

ranocha reviewed (commented):
Thanks. Could you please update the `README.md` in the root directory, too?

andrewwinters5000 commented:
> Thanks. Could you please update the `README.md` in the root directory, too?

Done. I updated the title in both references, the abstract, and added information about the additional `code_curved` folder.

ranocha reviewed (approved):
Thanks a lot!
